.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
+  "env": {
+    "node": true,
+  },
+  "extends": "amo",
```

Comment:
not important, I just feel that `amo` gives not enough context to people who have never seen that term before.